runtime.md

```diff
@@ -15,3 +15,46 @@ Runs a process in a container. Can be invoked several times.
 Not sure we need that from oc cli. Process is killed from the outside.
 
 This event needs to be captured by oc to run onstop event handlers.
+
+## Hooks
+Hooks allow one to run code before/after various lifecycle events of the container. The state of the container is passed to the hooks over stdin, so the hooks could get the information they need to do their work.
+
+The path to the hooks could be absolute or just the name of the file. Hooks are executed from the host's filesystem. If the path to a hook isn't absolute then it is searched for within the PATH environment variable on the host.
```

Comment:
On Wed, Jul 22, 2015 at 04:47:37PM -0700, W. Trevor King wrote:
> Ah, I was thinking that we might have fat containers that are
> simultaneously compatible with several OSes.

And thinking this over some more, I guess we could still be shooting
for this goal, just via several per-platform configs, instead of a
single, multi-platform config:

  .
  ├── config-linux.json
  ├── config-os-x.json
  ├── config-windows.json
  └── rootfs
      ├── linux
      ├── os-x
      └── windows

The current path handling is fine, but I thought I'd post this
follow-up comment to avoid spreading my earlier confusion about if/how
fat containers would be supported.